### PR TITLE
Add CMake version check for /showIncludes workaround

### PR DESCRIPTION
**Pull request by tristanlabelle**
To avoid hardcoding `CMAKE_CL_SHOWINCLUDES_PREFIX` unless we need it since it can break builds on non-English machines.

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -330,6 +330,18 @@ function Append-FlagsDefine([hashtable]$Defines, [string]$Name, [string]$Value)
   }
 }
 
+function Test-CMakeMinVersion([int]$Major, [int]$Minor, [int]$Patch = 0) {
```
Comment (compnerd) [inline build.ps1:333]: Does `Test-CMakeAtLeast` fit the naming scheme for PS?  If so, that seems like a nicer name.

**Review by compnerd — APPROVED** [review]
Thanks for improving this!  I wanted this, but didn't want to futz with PS.

File: build.ps1 (hunk 1 above)
Comment (tristanlabelle) [reply build.ps1]: I don't know if it's legit but I took your suggestion.

**Comment by hjyamauchi** [comment]
Nice!

**Pull request merged**